include/wx/qt/menuitem.h

```diff
@@ -46,9 +43,12 @@ class WXDLLIMPEXP_CORE wxMenuItem : public wxMenuItemBase
     virtual void ClearExtraAccels() override;
 #endif // wxUSE_ACCEL
 
+    // implementation
+    void CreateQtAction(wxMenu* parentMenu);
```

Comment:
Very minor, but I'd rename this to `QtCreateAction()` for consistency with the other impl-only functions that use port-specific prefixes.